.github/workflows/deploy-benchmark.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: Benchmark Service
+
+on:
```

Comment:
Let's also include paths so it doesn't get triggered when someone changes, e.g., the snapshot service.

Adding concurrency would also be helpful. Example here https://github.com/ChainSafe/forest-iac/blob/82884863aac0b04e4c9c55a4024564bdad0984e7/.github/workflows/deploy-forest-calibnet.yml#L4-L17

Reply:
done

Reply:
What about concurrency?